src/styles/stitches.config.ts

```diff
@@ -54,6 +63,20 @@ export const { styled, css, globalCss, keyframes, getCssText, theme, createTheme
         paddingX: (value: string) => ({ paddingLeft: value, paddingRight: value }),
         paddingY: (value: string) => ({ paddingTop: value, paddingBottom: value }),
         boxSizingForAllChildren: (value: string) => ({ '*': { boxSizing: value } }),
+        elevationBoxShadow: (value: TElevationType) => {
+            if (!Array.isArray(elevation_box_shadow[`${value}`])) {
+                const { x, y, blur, spread, color } = elevation_box_shadow[`${value}`] as TBoxShadowProperties;
+                return {
+                    boxShadow: `${x} ${y} ${blur} ${spread} ${color}`,
+                };
+            }
+
+            const box_shadow_array = (elevation_box_shadow[`${value}`] as TBoxShadowProperties[]).map(
```

Comment:
Will this ever be an array? 

```
export const elevation_box_shadow = {
    xs :TOKEN.General['xs'].value,
    sm :TOKEN.General['sm'].value,
    md :TOKEN.General['md'].value,
    lg :TOKEN.General['lg'].value,
    xl :TOKEN.General['xl'].value,
    xxl :TOKEN.General['xxl'].value,
    xxxl :TOKEN.General['xxxl'].value,
}
```
🤔 when u access by a value key, wouldn't it be just a single object?

Reply:
yes, some of the values are array